src/python/foglamp/storage/payload_builder.py

```diff
@@ -0,0 +1,217 @@
+        condition = {}
+        if self.verify_condition(arg):
+            condition.update({"column": arg[0], "condition": arg[1], "value": arg[2]})
+            self.payload.update({"where": condition})
+        return self
+
+    def WHERE_AND(self, *args):
+        for arg in args:
+            condition = {}
+            if self.verify_condition(arg):
+                condition.update({"column": arg[0], "condition": arg[1], "value": arg[2]})
+                self.payload["where"].update({"and": condition})
+        return self
+
+    def WHERE_OR(self, *args):
+        for arg in args:
+            if self.verify_condition(arg):
+                condition = {}
+                condition.update({"column": arg[0], "condition": arg[1], "value": arg[2]})
+                self.payload["where"].update({"or": condition})
+        return self
+
+    def GROUPBY(self, *args):
+        self.payload.update({"group": ', '.join(args)})
+        return self
+
+    def AGGREGATION(self, *args):
+        for arg in args:
+            aggregation = {}
+            if self.verify_aggregation(arg):
+                aggregation.update({"operation": arg[0], "column": arg[1]})
+                if 'aggregation' in self.payload:
+                    if isinstance(self.payload['aggregation'], list):
+                        self.payload['aggregation'].append(aggregation)
+                    else:
+                        self.payload['aggregation'] = list(self.payload.get('aggregation'))
+                        self.payload['aggregation'].append(aggregation)
+                else:
+                    self.payload.update({"aggregation": aggregation})
+        return self
+
+    def HAVING(self):
+        # TODO: To be implemented
+        return self
+
+    def LIMIT(self, arg):
+        if isinstance(arg, int):
+            self.payload.update({"limit": arg})
+        return self
+
+    def ORDERBY(self, *args):
+        for arg in args:
+            sort = {}
+            if self.verify_orderby(arg):
+                sort.update({"column": arg[0], "direction": arg[1]})
+                if 'sort' in self.payload:
+                    if isinstance(self.payload['sort'], list):
+                        self.payload['sort'].append(sort)
+                    else:
+                        self.payload['sort'] = list(self.payload.get('sort'))
+                        self.payload['sort'].append(sort)
+                else:
+                    self.payload.update({"sort": sort})
+        return self
+
+    def execute(self):
+        return dict(self.payload)
+
+    def execute_query_params(self):
+        where = self.payload['where']
+        query_params = {where['column']: where['value']}
+        for key, value in where.items():
+            if key == 'and':
+                query_params.update({value['column']: value['value']})
+        return urllib.parse.urlencode(query_params)
+
+if __name__ == "__main__":
+    pb = PayloadBuilder(payload=OrderedDict())
```

Comment:
1) better if don;t need to make it an object (should optional)
2) pb = PayloadBuilder() will work too, yes? as payload=OrderedDict() is default arg